### ISSUE TITLE
Sometimes maven doesn't find the plugin

### ISSUE THREAD

**Issue by raboof** [akka/akka-grpc#226]
https://travis-ci.org/akka/akka-grpc/jobs/386610582

**Comment by johanandren** [comment]
I was just wondering about that failure ...

**Comment by johanandren** [comment]
Could it be because of:
```
The command "sbt publishM2 && cd plugin-tester-java && mvn -Dakka.grpc.project.version=`git describe --tags | sed -e "s/v\([0-9]*.[0-9]*\)-\([0-9]*\).*/\1+\2-/"``echo $TRAVIS_COMMIT | head -c8` akka-grpc:generate compile" exited with 1.
```

**Comment by johanandren** [comment]
Ach, that is the entire command that is failing right, not a build step. NVM

**Comment by raboof** [comment]
fixed in #248
